.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+. "$(dirname "$0")/_/husky.sh"
+
+yarn run lint && yarn test
```

Comment:
It would be useful to have Husky fix linting errors when possible and commit the fixes (e.g., [this article](https://codeburst.io/continuous-integration-lint-staged-husky-pre-commit-hook-test-setup-47f8172924fc) on using `lint-staged`). What do you think?

I also saw a suggestion to run only affected tests, like using Jest's [--changedSince=master](https://jestjs.io/docs/cli#--changedsince) or [--onlyChanged](https://jestjs.io/docs/cli#--onlychanged) flags, which sounds great. I'm a little skeptical of the overhead of running the whole test suite on commit.

Reply:
This is a great idea - I'm currently struggling getting any flags (--changedSince, --onlyChanged and --findRelatedTests) to pick up, well, any tests, which is slowing down my velocity on this somewhat.